docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
-A:  User Sync is an open source python project.  Users can build for any OS platform they desire.  We provide builds for Windows, OS X, Ubuntu, and Cent OS 7 platforms.
+### The user sync tool is dedicated to a particular OS? (for example, win package 
+is different than unix or linux installer?)
+
+User Sync is an open source python project.  Users can build for any OS platform they desire.  We provide builds for Windows, OS X, Ubuntu, and Cent OS 7 platforms.
  
-**Q: Has this been tested on python 3.5?**<br/>
-A: User Sync has been run successfully on Python 3.x, but most of our use and testing is on Python 2.7 so you may discover problems, and we only provide builds on Python 2.7.  Feel free to report (and contribute fixes) to the open source site at https://github.com/adobe-apiplatform/user-sync.py.
+### Has this been tested on python 3.5?
 
-**Q: If something changes in the API (new field in creating users, for example ) how will the update  be applied to the user sync tool?** New package or python modules can be directly modified customer side?<br/>
-A: User sync is an open source project.  Users can download and build the latest 
+User Sync has been run successfully on Python 3.x, but most of our use and testing is on Python 2.7 so you may discover problems, and we only provide builds on Python 2.7.  Feel free to report (and contribute fixes) to the open source site at https://github.com/adobe-apiplatform/user-sync.py.
+
+### If something changes in the API (new field in creating users, for example ) how will the update  be applied to the user sync tool?
+
+User sync is an open source project.  Users can download and build the latest 
 sources at their discretion.  Adobe will post new releases with builds periodically.  
 Users can stay informed of them via git notifications.  When adopting a new release, 
 only the single pex file needs to be updated by the user.  If there are configuration 
 changes or command line changes to support new features, there may be updates in 
 those files to take advantage of them.
 
-**Q: Does User sync need some sort of whitelisting with the firewall rules of the machine
-on which it runs?** I see Windows users are more prone to such errors, from support 
-experience.<br/>
-A: User sync needs to be able to access the enterprise directory system and Adobe.  
+### Does User sync need some sort of whitelisting with the firewall rules of the machine
+on which it runs? I see Windows users are more prone to such errors, from support 
+experience.
+
+User sync needs to be able to access the enterprise directory system and Adobe.  
 Since there can be significant variance across system setups, we can't give any 
 specific steps.  LDAP typically uses port 389, but can be configured by IT.  Adobe 
 only requires REST calls over SSL on port 443.
 
-**Q: There is something specified about a second step in differentiating the users 
+### There is something specified about a second step in differentiating the users 
```

Comment:
broken question, and the question overall is completely unintelligible.  you need to rewrite it.